lib/scarpe/wv/video.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class WebviewVideo < Scarpe::WebviewWidget
+    def initialize(properties)
+      @url = properties[:url]
+      super
+    end
+
+    def element
+      HTML.render do |h|
+        h.video(id: html_id, style: style, controls: true) do
+          h.source(src: @url, type: "video/mp4")
```

Comment:
If we only support mp4, we should check that in the Shoes-side widget and give an error if they try to use something without an mp4-ish extension.

At some point we should figure out which formats are possible for which webview platforms.

Reply:
hm is dng smthng like ok?

```
    def get_type
      file_extension = File.extname(@url).downcase
      case file_extension
      when '.mp4'
        'video/mp4'
      when '.webp'
        'video/webp'
      else
        'video/mp4' # hm default?
      end
    end
    ```

Reply:
or maybe maybe we can use mime this but i think its overkill

Reply:
oo or this? https://docs.ruby-lang.org/en/2.2.0/WEBrick/HTTPUtils.html

Reply:
I think the first bit with get_type will be fantastic. There's plenty of time to overengineer later, once it's clear where to do it :-)